demos/appyx-navigation/src/main/kotlin/com/bumble/appyx/navigation/MainActivity.kt

```diff
@@ -30,7 +34,14 @@ class MainActivity : NodeActivity() {
                 // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colorScheme.background) {
                     Column {
-                        NodeHost(integrationPoint = appyxIntegrationPoint) {
+                        NodeHost(
```

Comment:
we could have overloaded Android version of this where we automatically provide screen size from LocalConfiguration